appside/frames/body.js

```diff
@@ -215,15 +218,39 @@ class BodyMapFrame extends Frame {
             $(label).text(stmt);
             right.appendChild(label);
```

Comment:
Tangent: I noticed that these labels are missing the "for" attribute that ties the label to the checkbox. This attribute would enable the user to click the text to toggle the checkbox. See `ListBodyFrame.render` for an example of how to set the "for" attribute.

Reply:
Fixed 👍